src/ui/public/courier/fetch/request/request_fetch_params_to_body_provider.js

```diff
@@ -0,0 +1,15 @@
+import { requestFetchParamsToBody } from './request_fetch_params_to_body';
+
+export function RequestFetchParamsToBodyProvider(Promise, timefilter, kbnIndex, sessionId) {
+  const timeBounds = timefilter.getActiveBounds();
+  const indexToListMapping = {};
```

Comment:
These are being cached for the same duration as the `timeBounds` mentioned above.

Reply:
Great call on this, don't know why I stripped it out!